File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.java
class Solution {
    public int triangularSum(int[] nums) {
        int ans=0;
        int k=nums.length;
        while(k!=1){
            for(int i=0;i<k-1;i++){
                nums[i]=(nums[i]+nums[i+1])%10;
            }
            k-=1;
        }
        return nums[0];
    }
}

File: 2324-find-triangular-sum-of-an-array/find-triangular-sum-of-an-array.py
class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        k=len(nums)
        while k!=1:
            for i in range(k-1):
                nums[i]=(nums[i]+nums[i+1])%10
            k-=1
        return nums[0]